docs/ADD_NEW_CATALOG.md

```diff
@@ -0,0 +1,20 @@
+# Adding New Catalog to Hub
+
+This doc defines the steps to add a new catalog in Hub. The catalog **must** follow the structure defined in the [Catalog Organization TEP][tep].
+
+
+Process to add a new catalog:
+
+- Create a pull request to Hub repository adding your catalog details in [Hub Config file][config].
+-  Make sure you give a unique name which is not used for other catalogs defined in Config file.
+    This name will be used in identifying catalog and will be used in API to search resources.
+ eg.`/resource/<catalog-name>/<resource-name>`
+ 
+Once the pull request is reviewed and merged, the Next Steps are to be performed by the Hub maintainers.
+    
+- Invoke the `/system/config/refresh` API. This will add the new catalog details in db. To access the API, you need to have `config:refresh` scope. 
+- Now, use the Catalog refresh API `catalog/<catalogName>/refresh` to add resources from catalog in hub database. To access the API, you need to have `catalog:refresh` scope.
+    You can check on Hub UI for the new resources.
```

Comment:
should the owner of the new catalog be given scopes so that they can refresh their catalog whenever they want and not depend on the hub maintainers? :thinking:

Reply:
yeah he can add himself in hub config but he will get access to refresh any catalog as per current implementationion

Reply:
we will have to define a user per catalog who can refresh it